### DISCUSSION TITLE
RDC Files show but are missing randomized, large chunks of the google Earth Mesh

### DISCUSSION THREAD

**Discussion by Jack-Ironside** [eliemichel/MapsModelsImporter#296] [Q&A]
I am starting this discussion as I have commented over on the issue #275 but need answers as quickly as possible and I am at a complete loss.

The program is fantastic and has become such an important part of my workflow. As both I and @Superfreak1000 have tried seemingly all available versions and combinations I don't believe this is an issue of incompatibility.  MMImporter will occasionally bring in full sections of tiles and other times just random sections or a capture that's missing all of the central tiles. I also had it working in the past using older versions primarily Maps Model Importer 0.5.0 and Render Doc 1.9 which seemed very stable to me but its only started acting up recently.

I believe it has something to do with the distance the captures are taken from as that seems to be the only thing that changes the outcomes where a distance of around 550 meters seems to work best but is still not optimal and the blocks that are missed change every time. 

400 M

![BlenderView_TroubleShooting_400M](https://github.com/eliemichel/MapsModelsImporter/assets/44151972/96f67bc6-35e2-4fd4-8532-24a9f352c075)

500 M

![BlenderView_TroubleShooting_500M](https://github.com/eliemichel/MapsModelsImporter/assets/44151972/faa6bdeb-67f1-4def-9acd-414effa90286)

600 M

![BlenderView_TroubleShooting_600M](https://github.com/eliemichel/MapsModelsImporter/assets/44151972/ac1bbbc2-6bad-4919-aef6-147e7d682aaf)

I am unsure what this means but I know it isn't an issue with the "missing blocks" in the blender import settings as that doesn't change the outcomes at all. Beyond that the UI of Google Earth looks different? Maybe Google updated their API's or Google Earth in its own right, changing versions of blender, render doc, maps model importer and google chrome still have not made much changes and its hard to seemingly isolate the problematic application or step.

I have now come full circle and have reinstalled everything according to @eliemichel's Maps Models Importer v0.6.2 specifications (Blender 3.4, Renderdoc 1.25, and MMImporter 0.6.2), and am still suffering from similar accuracy issues.

Chrome View (550 M)

![ChromeView_TroubleShooting_550M](https://github.com/eliemichel/MapsModelsImporter/assets/44151972/642ce2c2-e489-4cdd-a8c6-cd4305989b87)

Blender View (550 M)

![BlenderView_TroubleShooting_550M](https://github.com/eliemichel/MapsModelsImporter/assets/44151972/01f52a8b-8695-4786-b38c-3550a9fcb263)

I still have a funny feeling that this has to do with google earth and render doc interacting with each other and is not an issue of MMImporter but I don't really know anyother software that reads .rdc files so I can't cancel out one or the other. The RDC file for the above capture is hosted here:

https://drive.google.com/drive/folders/1x0H9PkcEEkBLpSWO8WcFx2LPbv_GZZo-?usp=sharing

My PC specs are as follows:

PC Specs:

Device name DESKTOP-2CF20FH
Processor Intel(R) Core(TM) i9-10900F CPU @ 2.80GHz 2.81 GHz
Installed RAM 32.0 GB (31.9 GB usable)
Device ID 253F6B94-91AF-4396-BAC6-4FB081D42529
Product ID 00330-50000-00000-AAOEM
System type 64-bit operating system, x64-based processor
Pen and touch Pen support

Graphics Card:

NVIDIA GeForce RTX 3070
Driver Version: 31.0.15.3742

Windows Specs:

Edition Windows 10 Pro
Version 22H2
Installed on ‎2021-‎08-‎16
OS build 19045.3448
Experience Windows Feature Experience Pack 1000.19044.1000.0

**Comment by Jack-Ironside** [comment]
Ok so i have interestingly tried with google maps instead of google earth and it is not longer missing chunks. I believe my suspicions were right there is something with the new google earth that causes this issue. For me the view distance changing the outcomes completely must have to do with google earths max and min render distance? Or maybe the distance at which the Mesh chunks load in. I'm still not sure in this regard but @eliemichel might be something to look into? I'm happy with just using google maps for now but am curious what this issue is caused by.

**Comment by catalara99** [comment]
Hello everyone! I’ve been trying for 3 days to get this wonderful tool to work, but I’m not succeeding at all! I would like to solve my problem, but my time is running out and I really need to export a 3D image of a specific area from Google Maps to Sketchup.

I watched a lot of videos, read and re-read these forums, tested all versions of MapsModelsImporter, Blender and RenderDoc. But I was unsuccessful! Can anyone generate this map for me? It would be essential for my course completion work.

The area of my interest is in Vila Velha, Espírito Santo, Brazil. I will leave a print with the desired cut highlighting the points of interest. I will be eternally grateful to anyone who can help me, capturing this 3D map with textures and sending it to Blender and exporting it to a format compatible with Sketchup

![Screenshot_2](https://github.com/eliemichel/MapsModelsImporter/assets/156985181/0cb20b1f-e0fd-4659-a9d0-a39f521f2e17)

https://www.google.com.br/maps/@-20.321665,-40.2928094,5307m/data=!3m1!1e3?entry=ttu

**Reply by catalara99** [reply]
contact: +5527999212271
talaricopt@gmail.com
